index.html

```diff
@@ -88,7 +88,7 @@
         <h3>Freedom</h3>
 
         <details>
-          <summary>Don't get locked into walled gardens. Run any app you want -- even upload your own. You call the shots.</summary>
+          <summary>Sandstorm is free, open source software. Run any app you want -- even upload your own. Don't get locked into walled gardens.</summary>
```

Comment:
I don't think this hunk is needed anymore, now that the huge top text says it's open source.